src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,15 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.Future;
+import javax.validation.constraints.Min;
 
 public class MovieSessionRequestDto {
+    @Min(1)
```

Comment:
can we use NotNull?

Reply:
At the same time as @Min? I thought if we use @Min it tells us that it can't be null either, that is, in any case there will be a number greater than a certain value!